pkg/config/validation.go

```diff
@@ -0,0 +1,347 @@
+package config
+
+import (
+	"crypto/x509"
+	"errors"
+	"fmt"
+	"net/url"
+
+	"github.com/ghodss/yaml"
+	"github.com/loft-sh/vcluster/config"
+	"github.com/loft-sh/vcluster/pkg/util/toleration"
+	"github.com/samber/lo"
+	admissionregistrationv1 "k8s.io/api/admissionregistration/v1"
+)
+
+var allowedPodSecurityStandards = map[string]bool{
+	"privileged": true,
+	"baseline":   true,
+	"restricted": true,
+}
```

Comment:
you can add enums with jsonschema using tags

Reply:
We need this within golang code as well